### DISCUSSION TITLE
mircrotask-extension-math throw "Cannot find module 'assert'" error when NODE_ENV is 'development'

### DISCUSSION THREAD

**Discussion by curly210102** [micromark/micromark#89] [Q&A] [answered]
<img width="729" alt="Screen Shot 2021-09-12 at 1 30 12 AM" src="https://user-images.githubusercontent.com/76864176/132966955-a2f0b6a0-b99d-45a2-8c82-bdeb1551f307.png">

My project dependent remark-math, when project works in development mode, NODE_ENV is set as "development". But mircrotask-extension-math will run dev/index.js in development, https://github.com/micromark/micromark-extension-math/blob/main/package.json.

dev/index.js without unassert by microtask-build, is there any good solution rather than fallback to old version?

**Comment by wooorm** [comment, marked as answer]
See https://github.com/micromark/micromark/issues/87.

Quick answer: `npm install assert` or configure your bundler.
